graylog2-web-interface/src/pages/ShowDashboardPage.jsx

```diff
@@ -131,8 +134,8 @@ const ShowDashboardPage = React.createClass({
       positions[widget.id] = {
         col: (persistedDimensions.col === undefined ? defaultDimensions.col : persistedDimensions.col),
         row: (persistedDimensions.row === undefined ? defaultDimensions.row : persistedDimensions.row),
-        height: (persistedDimensions.height === undefined ? defaultDimensions.height : persistedDimensions.height),
-        width: (persistedDimensions.width === undefined ? defaultDimensions.width : persistedDimensions.width),
+        height: (this._validDimension(persistedDimensions.height) ? persistedDimensions.height : defaultDimensions.height),
```

Comment:
Yes, but it does not make the intention clearer. We use this shortcut all over the place when we can't be bother to look if it's `undefined` or `null`. Using `||` also checks for `false` and probably two or three other things. In this way I'm clearly stating that this value can be `0` even if it doesn't look like it.